designsafe/static/scripts/projects/components/manage-simulations/manage-simulations.component.js

```diff
@@ -1,4 +1,5 @@
 import ManageSimulationTemplate from './manage-simulations.component.html';
+const simulationTypes = require('./simulation-types.json');
```

Comment:
Can this be an ES6 import? Or does this require a JSON loader in our webpack configuration?